src/scenarios/personalization/ArrayGetFavorites/ArrayGetFavorites.tsx

```diff
@@ -0,0 +1,119 @@
+import {
+  useScenarioHost,
+  VariableSearch,
+} from '@axinom/mosaic-fe-samples-host';
+import { useState } from 'react';
+import {
+  Container,
+  Divider,
+  Form,
+  Grid,
+  Header,
+  Label,
+  Segment,
+} from 'semantic-ui-react';
+import { getApolloClient } from '../../../apollo-client';
+import { getArrayQuery } from './graphql-documents';
+
+export const ArrayGetFavorites: React.FC = () => {
+  const { activeProfile, logger } = useScenarioHost();
+  const [userAccessToken, setUserAccessToken] = useState<string>();
+
+  const fetchFavorites = async (): Promise<void> => {
+    try {
+      const apolloClient = getApolloClient(
+        new URL('graphql', activeProfile.personalizationServiceBaseURL).href,
+      );
+
+      const result = await apolloClient.mutate({
+        mutation: getArrayQuery,
+        variables: {
+          input: {
+            scope: 'PROFILE',
+            key: 'user_profile:favorites',
+          },
+        },
+        context: {
+          headers: {
+            Authorization: `Bearer ${userAccessToken}`,
+          },
+        },
+        fetchPolicy: 'no-cache',
+      });
+
+      if (result.errors) {
+        logger.error(
+          `method [${fetchFavorites.name}]`,
+          'output:',
+          result.errors,
+        );
+      } else {
+        logger.log(`method [${fetchFavorites.name}]`, 'output:', result.data);
+      }
+    } catch (error) {
+      if (error instanceof Error) {
+        logger.error(
+          `method [${fetchFavorites.name}]`,
+          'output:',
+          error.message,
+        );
+      }
+    }
+  };
+
+  return (
+    <Segment basic>
+      <Header size="huge">Array: Get Favorite(s)</Header>
```

Comment:
Update this depending on which version you decide to use - Favorite(s) or Favorites